tox.ini

```diff
@@ -4,12 +4,13 @@ skip_missing_interpreters = True
 
 [testenv]
 setenv =
+    PYTHONPATH = {toxinidir}:{toxinidir}/homeassistant
 ; both temper-python and XBee modules have utf8 in their README files
 ; which get read in from setup.py. If we don't force our locale to a
 ; utf8 one, tox's env is reset. And the install of these 2 packages
 ; fail.
-    LANG=C.UTF-8
-    PYTHONPATH = {toxinidir}:{toxinidir}/homeassistant
+whitelist_externals = /usr/bin/env
+install_command = /usr/bin/env LANG=C.UTF-8 pip install {opts} {packages}
```

Comment:
This install command is not correct. By default pip will point at Python 2. We should use `python3 -m pip` instead